lib/client.c

```diff
@@ -456,6 +456,7 @@ _snowflake_check_connection_parameters(SF_CONNECT *sf) {
     log_debug("timezone: %s", sf->timezone);
     log_debug("login_timeout: %d", sf->login_timeout);
     log_debug("network_timeout: %d", sf->network_timeout);
+    log_debug("snowflake_test_ca_bundle_file from getenv: %s", getenv("SNOWFLAKE_TEST_CA_BUNDLE_FILE"));
```

Comment:
This looks like a test debugging line, could this be moved out of client.c and into one of the test files maybe?